src/common/entropy-base.ts

```diff
@@ -5,7 +5,7 @@ export abstract class EntropyBase {
   protected logger: EntropyLogger
   protected entropy: Entropy
 
-  constructor (entropy: Entropy, endpoint: string, flowContext: string) {
+  constructor ({ entropy, endpoint, flowContext }: { entropy?: Entropy, endpoint: string, flowContext: string }) {
```

Comment:
why is `entropy` optional? I thought we dropped need for that when we started using `static` methods for `EntropyAccount`

```suggestion
  constructor ({ entropy, endpoint, flowContext }: { entropy: Entropy, endpoint: string, flowContext: string }) {
```

Reply:
BOOP

Reply:
done in another PR